app/workers/send_email_worker.rb

```diff
@@ -54,8 +54,6 @@ def increment_rate_limiter
   end
 
   def rate_limiter
-    @rate_limiter ||= Sidekiq.redis do |redis|
-      Ratelimit.new("send_email", redis: redis)
```

Comment:
The main difference here appears to be that the `Services` version is memoised at the class level. Could we try that and see if it fixes things?

Reply:
Yeah, I think that would have the same effect. It seems a bit of a smell though as it still has the same thread safety issues as Services and the oddness of using a distinct version of Redis to what Sidekiq uses.

Reply:
Yup, good idea.